src/main/java/pascal/taie/analysis/pta/plugin/exception/ExceptionAnalysis.java

```diff
@@ -218,6 +218,7 @@ private Set<CSObj> analyzeIntraUncaught(
             Stmt currentStmt,
             Set<CSObj> newExceptions,
             CSMethod csMethod) {
+        if (!catchers.containsKey(csMethod.getMethod())) return newExceptions;
         List<ExceptionEntry> entries = catchers.get(csMethod.getMethod())
```

Comment:
Thank you for your PR.

Although we currently lack a reproducible minimal test case, I believe this fix is intuitive.

The change I recommend eliminates the need to think through why we directly `return newExceptions`.

```suggestion
        List<ExceptionEntry> entries = catchers.getOrDefault(csMethod.getMethod(), Map.of())
```